languages/python/django-oso/django_oso/models.py

```diff
@@ -32,7 +33,10 @@ def authorize(self, request, *, actor=None, action=None):
 
         # SELECT DISTINCT on inner query to support chaining methods on
         # returned QuerySet.
-        return self.filter(pk__in=self.filter(filter).distinct())
+        if filter == TRUE_FILTER:
+            return self.filter(filter)
```

Comment:
If it's just the `TRUE_FILTER`, what's the point of calling `self.filter` at all?

Reply:
I think it needs to be something, and I didn't want to return `.all()` in case it overwrites previous filters. And this gets dropped in the SQL anyway.